raiden/network/proxies/payment_channel.py

```diff
@@ -42,6 +42,7 @@ def __init__(
             token_network=token_network,
             channel_identifier=channel_identifier,
             contract_manager=contract_manager,
+            from_block=token_network.metadata.filters_start_at,
```

Comment:
This is part of the fix. It will use the deploy block of `TokenNetworkRegistry` instead of the genesis block.